RELEASE_PROCESS.md

```diff
@@ -23,10 +23,10 @@ Users with permissions in the `@spectrum-web-components` organization on NPM can
 
 The docs site will publish automatically if the `#publish` string is included in the commit message and the check suite runs successfully.
 
-If publishing fails with an error, check the [list of tags](https://github.com/adobe/spectrum-web-components/tags) to see if new tags have been released for your publishing attempt. If they have, then `yarn lerna-publish` will no longer work. Instead, you'll need to run:
+If publishing fails with an error, check the [list of tags](https://github.com/adobe/spectrum-web-components/tags) to see if new tags have been released for your publishing attempt. If they have, then `yarn changeset-publish` will no longer work. Instead, you'll need to run:
 
 ```
-yarn lerna-publish from-package
+yarn changeset-publish from-package
```

Comment:
@blunteshwar is `from-package` still a valid arg that can be passed to `changeset-publish`?

Reply:
@blunteshwar @pfulton from what I understand, `from-package` is not a valid arg and [`yarn changeset-publish`](https://changesets-docs.vercel.app/en/command-line-options#publish) already does what [`yarn lerna-publish from-package`](https://lerna.js.org/docs/features/version-and-publish#from-package) does.